hipe4ml/model_handler.py

```diff
@@ -145,13 +144,16 @@ def fit(self, x_train, y_train):
 
         y_train: array-like, sparse matrix
             Target data
+
+        **kwargs:
+            Extra kwargs passed on to model.fit() method
         """
         n_classes = len(np.unique(y_train))
         self._n_classes = n_classes
         if self.training_columns is None:
             self.training_columns = list(x_train.columns)
 
-        self.model.fit(x_train[self.training_columns], y_train)
+        self.model.fit(x_train[self.training_columns], y_train, **kwargs)
 
     def predict(self, x_test, output_margin=True):
```

Comment:
Shall we have the `**kwargs` also here?

Reply:
Actually, it might get complicated with `train_test`... feel free to either think about it or ignore this.

Reply:
Thanks @mpuccio, done! But I would still keep train_test as it is